### ISSUE TITLE
Email form field on management page is not accessible

### ISSUE THREAD

**Issue by steverep** [stcr/subscribe-to-comments-reloaded#76]
The email form field on the virtual management page is not very accessible to screen readers.  The input field does not use the <label> tag, and for some reason the field is skipped when navigating the page in a "flat review" sense.  For the sake of completeness of my comment, I'm using the NVDA screen reader to test and looking at the page source.

**Comment by Reedyseth** [comment]
Hello @steverep 

I have downloaded the NVDA software and tested. The issue was that the `label` tag was missing the correct ID. This bug is fix on this commit 5a30ec8406f21430cfc060901848ff91edfca905.

I will mark this issue as closed but you are very welcome to open it if you find something else.

Regards.!!
